src/main/webapp/app/shared/components/confirm-autofocus-button.component.ts

```diff
@@ -44,6 +46,11 @@ export class ConfirmAutofocusButtonComponent {
         } else {
             modalRef.componentInstance.translateText = false;
         }
+        if (this.textIsMarkdown !== undefined) {
+            modalRef.componentInstance.textIsMarkdown = this.textIsMarkdown;
+        } else {
+            modalRef.componentInstance.textIsMarkdown = false;
+        }
```

Comment:
```suggestion
        modalRef.componentInstance.textIsMarkdown = !!this.textIsMarkdown;
```
I'm not sure but I think this could be shortend like shown here.

Reply:
Can't test this right now, but looks neat!